### DISCUSSION TITLE
measuring and tracking system temperatures

### DISCUSSION THREAD

**Discussion by proddy** [tteck/Proxmox#833] [General]
Hi, I've moved my Ubuntu/docker stack over to a proxmox on a small NUC8 and it's great. 

Every night around 4am I get a SMART warning in the Syslog saying my Samsung SSD 980 is overheating. I was wondering if there are any recommendations for showing cpu, system, gpu, hdisk etc temperatures without using a whole new app like netdata/glances/webmin etc?

Something like https://medium.com/help-14/add-amd-cpu-gpu-nvme-temperature-monitoring-to-proxmox-7-a95147df71cd is cool (no pun intended) but I'm not sure it's generic enough for all systems or even if hacking the proxmox dashboard will be overwritten when theses scripts update

**Comment by Ramalama2** [comment]
You could push the temperatures to grafana, using collectd and the thermal plugin.
And collectd pushes it to influxdb.
Then you make a graph or singlestat, whatever you like in grafana.

As there is no other way in proxmox, without hacking through and the changes going to be reverted every update, this way sounds at least for me as the best way 🙈

Btw:
And yeah, i understand that hosting influxd + grafana additionally only for this, is probably not that straightforward.
But see it positive, you can configure additionally the proxmox metrics and you gain at least more info to build graphs etc...
Grafana is relatively lightweight, i only suggest using influxdb2 over influxdb1, because of how much influxdb2 is more performant.
The difference is huge, but influxdb2 requires you to learn building graphs with the flux language instead of clicky through as the only downside 🙈
